### PR TITLE
bit-0.6.4

### PR DESCRIPTION
**Pull request by amitgilad3**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/bit.rb [hunk 1]
```diff
@@ -20,7 +20,6 @@ def install
   end
 
   test do
-    assert_equal "successfully initialized an empty bit scope.\n",
-                 shell_output("#{bin}/bit init --skip-update")
+    system bin/"bit", "init", "--skip-update"
```
Comment (dunn) [inline Formula/bit.rb]: Did the printed message change?

Comment (amitgilad3) [inline Formula/bit.rb]: Hi dunn,
yes the printed message did change, it is now depends on the language the user is working with.
so the message can change according to language.
thats why the only way to assure that bit is working is by running bit init and having it return a zero exit code.

**Comment by dunn** [comment]
@jcount why the label?

**Comment by JCount** [comment]
@dunn This is unnecessarily bundling a precompiled node binary, when it really should use the one that is part of Homebrew.

**Comment by JCount** [comment]
Whoops, smartphones.

**Comment by JCount** [comment]
As is, this also has at least one hard coded path that maked installs, whether from "source" or poured from a bottle, broken when using a prefix other than `/usr/local`.

**Comment by ilovezfs** [comment]
Any news here?

**Comment by ilovezfs** [comment]
@amitgilad3 any update here?

**Pull request closed without merging**